### PR TITLE
Toggle Theme Added

### PR DESCRIPTION
**Pull request by Saurav-Pant**
Here I added the toggle Theme functionality for the app, the demo is here 

https://user-images.githubusercontent.com/103263909/233842632-f334f5fd-0201-4bf9-84cb-7d715780bed6.mp4

### CODE REVIEW

File: src/pages/ProjectsPage.jsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ const ProjectsPage = () => {
       <h1 className="text-[3.5rem] font-bold  text-center">
         List of <span className="text-primary">cool </span>Projects
       </h1>
-      <p className="mt-3 text-gray-300 text-[1.2rem] text-center mx-auto w-10/12">
+      <p className="mt-3 text-[1.2rem] text-center mx-auto w-10/12">
```
Comment (priyankarpal) [inline src/pages/ProjectsPage.jsx:51]: Add `text-gray-300`

```suggestion
      <p className="mt-3 text-gray-300 text-[1.2rem] text-center mx-auto w-10/12">
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by priyankarpal** [comment]
ok looks good.

but you need to do some changes

1.  don't change bg colour 
2. Add icons beside the discord icon 
image
3. don't change the navbar color also
4. also change the size of the icon

**Comment by priyankarpal** [comment]
> ok looks good.
> 
> but you need to do some changes
> 
> 1. Don't change bg colour
> 2. Add icons beside the discord icon
>    image
> 3. Don't change the navbar colour also
> 4. Also change the size of the icon

check this

**Comment by Saurav-Pant** [comment]
1. "don't change bg color" What is this for?
2. I will do it 
3. I will do it 
4.I will do it 

> > ok looks good.
> > but you need to do some changes
> > 
> > 1. Don't change bg colour
> > 2. Add icons beside the discord icon
> >    image
> > 3. Don't change the navbar colour also
> > 4. Also change the size of the icon
> 
> check this

**Comment by Saurav-Pant** [comment]
1. "don't change bg color" What is this for?
2, 3, 4. I will do this.

**Comment by priyankarpal** [comment]
> > ok looks good.
> > but you need to do some changes
> > 
> > 1. Don't change bg colour
> > 2. Add icons beside the discord icon
> >    image
> > 3. Don't change the navbar colour also
> > 4. Also change the size of the icon
> 
> check this

background color for the website `#000513`
nav bar color `#111827`

also, add tailwind CSS not inline CSS.

**Comment by Saurav-Pant** [comment]
for both themes, Navbar color should same?

**Comment by priyankarpal** [comment]
> for both themes, Navbar color should same?

no for dark mode

**Comment by Saurav-Pant** [comment]
"nav bar color #111827
also, add tailwind CSS not inline CSS" how do I achieve this? suggest me

**Comment by priyankarpal** [comment]
> "nav bar colour #111827 also, add tailwind CSS not inline CSS" how do I achieve this? suggest me


only change this:
add toggle switch here & make it small size

image

**Comment by Saurav-Pant** [comment]
(https://user-images.githubusercontent.com/103263909/233847869-b01a22bc-bd1e-47df-a1ff-22fe08b56fad.png)

**Comment by priyankarpal** [comment]
> (https://user-images.githubusercontent.com/103263909/233847869-b01a22bc-bd1e-47df-a1ff-22fe08b56fad.png)

yes. cool

**Comment by Saurav-Pant** [comment]
ok, I think I did all changes which you said.

**Comment by priyankarpal** [comment]
> ok, I think I did all changes which you said.

I mean just use the navbar colour (previously said) only for dark mode not for light mode.  😢

**Comment by Saurav-Pant** [comment]
ok, i try this.

**Comment by priyankarpal** [comment]
> ok, i try this.

ok ok, take your time. it's not urgent  😅

**Comment by Saurav-Pant** [comment]
yes yes 😅

**Comment by Saurav-Pant** [comment]
how is this?
https://user-images.githubusercontent.com/103263909/233851676-14c0349a-b5d0-48d9-982c-df99dd9a5b83.mp4

**Comment by priyankarpal** [comment]
> how is this? https://user-images.githubusercontent.com/103263909/233851676-14c0349a-b5d0-48d9-982c-df99dd9a5b83.mp4

perfect

**Comment by priyankarpal** [comment]
ok just a few changes required  
change the card's texts colour to white according to the theme 

image

**Comment by Saurav-Pant** [comment]
For my device is working like this
https://user-images.githubusercontent.com/103263909/233852566-cbde2ada-60ac-466e-bdac-4d1d95b539c4.mp4

**Comment by priyankarpal** [comment]
> For my device is working like this https://user-images.githubusercontent.com/103263909/233852566-cbde2ada-60ac-466e-bdac-4d1d95b539c4.mp4

ok then change the menu bar color to black for light mode

**Comment by Saurav-Pant** [comment]
Changed

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**